packages/react-components/src/organisms/WorkingGroupsTabbedCard.tsx

```diff
@@ -0,0 +1,67 @@
+import { css } from '@emotion/react';
+import React from 'react';
+
+import { WorkingGroupMembership } from '@asap-hub/model';
+import { TabbedCard, WorkingGroupsList } from '../molecules';
+import { rem, tabletScreen } from '../pixels';
+import { splitListBy } from '../utils';
+
+const containerStyles = css({
+  padding: `${rem(32)} 0`,
+  display: 'grid',
+  rowGap: rem(12),
+  [`@media (min-width: ${tabletScreen.min}px)`]: {
+    columnGap: rem(15),
+  },
+});
+
+type WorkingGroupsTabbedCardProps = {
+  userName: string;
+  groups: ReadonlyArray<WorkingGroupMembership>;
+  isUserAlumni: boolean;
+};
+
+const WorkingGroupsTabbedCard: React.FC<WorkingGroupsTabbedCardProps> = ({
+  userName,
+  groups,
+  isUserAlumni,
+}) => {
+  const [inactiveMemberships, activeMemberships] = splitListBy(
```

Comment:
Hey, I was testing manually and then I saw this error
<img width="1440" alt="reduce" src="https://user-images.githubusercontent.com/16595804/211365859-a545dbfe-6042-46d1-b3eb-86796a27e006.png">

Looking at console it seems the problem is with the reduce in `splitListBy` function

<img width="1435" alt="console1" src="https://user-images.githubusercontent.com/16595804/211366339-749db57a-7947-4e9a-ac7a-59cdf2eda4d2.png">

<img width="1433" alt="console2" src="https://user-images.githubusercontent.com/16595804/211366359-f4d752b9-0f0f-4aed-9fc1-d9ba2c6bf5fb.png">


This function assumes that `groups` is never undefined, so while it's loading it gives this error.

I took a look at how is done for groups while it's loading and it seems that the whole section does not appear while the data hasn't arrived yet https://github.com/yldio/asap-hub/blob/master/apps/crn-frontend/src/network/users/groups/GroupsCard.tsx#L17

I would do the same here. Something like

```
if (!groups) return null
```
in line 29. But I would just check with Tiff if that's ok :)